topo/node/srl/srl.go

```diff
@@ -238,12 +271,13 @@ func (n *Node) Create(ctx context.Context) error {
 }
 
 func (n *Node) Delete(ctx context.Context) error {
-	err := n.ControllerClient.Delete(ctx, &srlinuxv1.Srlinux{ObjectMeta: metav1.ObjectMeta{Name: n.Name()}})
+	err := n.ControllerClient.Delete(ctx, &srlinuxv1.Srlinux{
+		ObjectMeta: metav1.ObjectMeta{
+			Namespace: n.GetNamespace(), Name: n.Name()}},
+	)
```

Comment:
When breaking up the line you should match the {}'s. 

```
        err := n.ControllerClient.Delete(ctx, &srlinuxv1.Srlinux{
                ObjectMeta: metav1.ObjectMeta{
                        Namespace: n.GetNamespace(), Name: n.Name(),
                },
        })
```

Reply:
thanks
Done in [9a561cd](https://github.com/openconfig/kne/pull/343/commits/9a561cd50b503c75366a246a4ff40e33ffb8bea3)